eZ/Publish/Core/MVC/Symfony/View/Builder/ContentViewBuilder.php

```diff
@@ -84,10 +84,10 @@ public function buildView(array $parameters)
             $view->setViewType(EmbedView::DEFAULT_VIEW_TYPE);
         }
 
-        if (isset($parameters['locationId'])) {
-            $location = $this->loadLocation($parameters['locationId']);
-        } elseif (isset($parameters['location'])) {
+        if (isset($parameters['location'])) {
```

Comment:
To be on a safe side:
```suggestion
        if (isset($parameters['location']) && $parameters['location'] instanceof Location) {
```